<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initWindow</class>
 <widget class="QMainWindow" name="initWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>麻将连连看</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>1600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/initbackground.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/title.png</pixmap>
    </property>
   </widget>
   <widget class="MyLabel" name="label_begin">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>580</y>
      <width>731</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/begin.png</pixmap>
    </property>
   </widget>
   <widget class="MyLabel" name="label_choosedifficulties">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>850</y>
      <width>741</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/choosedifficulties.png</pixmap>
    </property>
   </widget>
   <widget class="MyLabel" name="label_rule">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>1130</y>
      <width>741</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/rule.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
